site/src/sections/hero.style.js

```diff
@@ -0,0 +1,129 @@
+import styled from "styled-components";
+
+const HeroStyleWrapper = styled.div`
+    height: 53rem;
+    width: 100%;
+    background-color: #00B39F;
+    color: #FFF;
+    font-family: 'Qanelas Soft', 'Open Sans', sans-serif;
+    display: flex;
+    postion: relative;
+    overflow: hidden;
+    
+    .content {
+        position: absolute;
+        display: flex;
+        justify-content: space-between;
+        margin-inline: 10%;
+        margin-top: 6%;
+        @media(max-width: 950px) {
+            flex-direction: column;
+            align-items: center;
+            margin-inline: 0;
+            margin-top: 0%;
+            gap: 1rem;
+        }
+    }
+
+    .info {
+        width: 50%;
+        display: flex;
+        flex-direction: column;
+        gap: 1em;
+        margin-top: 4.5rem;
+        @media(max-width: 950px) {
+            width: 90%;
+        }
+
+        
+    }
+    
+    .title {
+        color: #fff;
+        font-size: 4.6rem;
+        margin-bottom: .5rem;
+        @media(max-width: 750px) {
+            font-size: 2.5rem;
+        }
+    }
+
+    .description {
+        color: #fff;
+        font-size: 1.9rem;
+        @media(max-width: 950px) {
+            font-size: 1.5rem;
+        }
+    }
+
+    .credit {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        gap: 1.25em;
+        
```

Comment:
and here.

Reply:
I don't think i can remove these, because the icons in the hero section become squished when I do.

Reply:
ok got it